src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -633,14 +633,14 @@ properties:
       - Smoking history not documented
       - Unknown
       - Not Reported
+      - Not Allowed To Collect
       - "1"
```

Comment:
Numberic should be placed ahead of the 'Unknown, Not Reported, Not Allowed to Collect' set. (or the documentation is updated)

Reply:
I agree, the property these enums is attached to is excused from the auto sorting since it has frequency text terms and I overlooked it when reviewing the changes for a manual fix. I will update it.